Formula/rtags.rb

```diff
@@ -28,6 +28,45 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
+    The instance of rtags that run via brew services listens to
+    the socket file at #{var}/.rdm
```

Comment:
> Hidden directories are preferred in HOME by users
Using the default value of `~/.rdm` would actually improve usability I think, since the simple
invocation of `rc` will work without having to specify `rc --socket-file=#{var}/rdm`.

But I'm not sure if this is good practice. Is there a policy or a convention for home-brew
packages around this?